third_party/java/arcs/build_defs/internal/manifest.bzl

```diff
@@ -41,27 +41,32 @@ def _generate_root_manifest_content(label, input_files):
     content = ["// Root manifest generated by %s." % label]
     for input_file in input_files:
         if input_file.basename.endswith(".arcs"):
-            content.append("import '%s'" % input_file.short_path)
+            content.append("import 'https://$assets/%s'" % input_file.short_path)
```

Comment:
I strongly think that the generated manifest should use relative paths instead of a macros. If you use a macro here then you require the runtime to have a loader with such macro configured.

If you want some interesting loading behavior, then you configure the loader for a macro and load $macro/path/to/manifest.arcs, and $macro covers manifest.arcs and everything that's important in a relative way.

Reply:
Done. I had to do it in the .bzl file, so the logic is kind of primitive. It now computes a ../../.. path back to the repo root, and then appends the full path to the imported .arcs file.

Reply:
I understand. It's important that it works :)